MusicApp.Identity.Domain/Exceptions/CommonExceptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Security.Authentication;
+
+namespace MusicApp.Identity.Domain.Exceptions;
+
+public static class CommonExceptions
+{
+    public static AlreadyExistsException usernameIsTaken = new("This username is taken.");
```

Comment:
Why?

Reply:
I want usernames in my application to be unique.

Reply:
it's a little bit strange idea, it's better to define custom exceptions with default error messages and throw these exceptions where necessary